tests/test_packages/test_connections/test_ledger/test_contract_api.py

```diff
@@ -70,13 +70,18 @@ def load_erc1155_contract():
     contract = Contract.from_config(configuration)
     assert contract.contract_interface is not None
 
+    # TODO some other tests don't deregister contracts from the registry.
+    #   find a neater solution.
+    if configuration.public_id in contract_registry.specs.keys():
+        contract_registry.specs.pop(str(configuration.public_id))
```

Comment:
I think we need this because the contract may be already registered by other tests, not necessarily the ones using this fixture

Reply:
(In particular, any in-process code that runs an AEA with the ERC1155 contract!)